recipes/pydash/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "pydash" %}
+{% set version = "4.2.1" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "58573ec724d696a2268fec6bcb4e19549cb15876ec03837c4455115f95712e6a" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
```

Comment:
I think this package is a candidate for `noarch: python`

http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages